### PR TITLE
Add Changelog for v0.1 Release

### PR DESCRIPTION
**Pull request by asifdxtreme**
https://github.com/volcano-sh/volcano/issues/67

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
+# Change Log
+
+## [v0.1](https://github.com/volcano-sh/volcano/tree/v0.1) (2019-05-11)
+**Implemented enhancements:**
+
+- Enable robot for Volcano [\#15](https://github.com/volcano-sh/volcano/issues/15)
```
Comment (k82cn) [inline CHANGELOG.md]: please also highlight other important features.

Comment (asifdxtreme) [reply CHANGELOG.md]: this release notes is auto-generated, it picks up enhancement based on the issue type we create in github issues

Comment (k82cn) [reply CHANGELOG.md]: we're using kind/feature for that, right? 

xref https://github.com/volcano-sh/volcano/issues/15#issuecomment-477409104

Comment (asifdxtreme) [reply CHANGELOG.md]: @k82cn yes it seems like it picks from that, but looks like kind/feature is not supported by https://github.com/github-changelog-generator/github-changelog-generator

Comment (k82cn) [reply CHANGELOG.md]: so what's our next step?

Comment (asifdxtreme) [reply CHANGELOG.md]: we can remove this section, this sections looks like a contradictory one

**Pull request merged**